Invert implementations to run/stop oscillator (#3)
eldruin/ds1307-rs: CHANGELOG.md

- Updated internal use of `chrono`.
- Updated `embedded-hal-mock` dev-dependency.
- Updated MSRV to 1.62.0.

> - [breaking-change] Swap implementations of `set_running` and `halt`, inverted output of `running` as CH bit was interpreted incorrectly.

Reviewer: ```suggestion

### Fixed
- Swapped implementations of `set_running` and `halt` and fixed output level of `running`.
```
